plugins/woocommerce-beta-tester/api/api.php

```diff
@@ -66,3 +66,4 @@ function () use ( $route, $callback, $additional_options ) {
 require 'remote-inbox-notifications/class-wca-test-helper-remote-inbox-notifications.php';
 require 'remote-logging/remote-logging.php';
 require 'tools/wccom-request-errors.php';
+require 'tools/set-woocom-base-url.php';
```

Comment:
Nit: we have prior art with `wccom-request-errors.php`, I think it'll be slightly better to standardize - otherwise, new devs may think it's two different things
```suggestion
require 'tools/set-wccom-base-url.php';
```